ui/src/views/Documentation/Reference/SchemaIndex.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component, Fragment } from 'react';
+import { withRouter } from 'react-router-dom';
+import sortBy from 'lodash/sortBy';
+import filter from 'lodash/filter';
```

Comment:
Let's switch to using ramda, which is what we use in the ui directory. You can probably find the sort and filter methods in https://ramdajs.com/docs/.